jupyterlab_gitsync/src/components/toolbar.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import { classes } from 'typestyle';
+
+import {
+  ControlButton,
+  SaveButton,
+  SyncButton,
+  ReloadButton,
+} from './toolbar_buttons';
+
+import { toolbarClass } from '../style/toolbar';
+
+import { Props, State } from './panel';
+
+export class Toolbar extends React.Component<Props, State> {
+  /**
+   * Returns a React component for rendering a panel toolbar.
+   *
+   * @param props - component properties
+   * @returns React component
+   */
+  constructor(props: Props) {
+    super(props);
+
+    this.state = {
+      state: 'Running',
```

Comment:
Is this state used anywhere?

We should send the frontend part of the PR to also our frontend team, probably prodonjs@

Reply:
I removed all necessary state attributes to UI components